tools/dev/credit.py

```diff
@@ -0,0 +1,65 @@
+"""TODO: Maybe remove. Not currently used because it's too slow."""
+
+import os
+import re
+from tqdm import tqdm
+from pathlib import Path
+from github import Github
+from github import Auth
+
+ignores = [
+    1420,  # Split huge viz.py to a proper module
+    5862,  # MAINT: splitting utils.py to submodules
+    6767,  # MRG, DOC: Integrate manual parts
+    11667,  # MAINT: Run black on codebase
+    11868,  # MAINT: Fix YAML spacing
+    12097,  # Sort imports
+    12261,  # MRG: Use ruff-format instead of Black
+    12533,  # MAINT: Clean up PyVista contexts
+    12588,  # [pre-commit.ci] pre-commit autoupdate
+    12603,  # STY: Apply ruff/pyupgrade rule UP028
+]
```

Comment:
really nice work here, thanks!

would it make sense to use https://github.com/mne-tools/mne-python/blob/main/.git-blame-ignore-revs for this?

Reply:
Yes in principle! I actually do use the `.git-blame-revs-ignore` in the `.sh` file. There is probably a way to map the commit numbers in the ignore file back to the PRs they came from, which would allow this script to do some similar ignoring. I'll look into it if I come back to using this script -- right now everything is done in the bash script because doing it in Python is too slow!